### ISSUE TITLE
Terminal label distance with left-aligned tabs

### ISSUE THREAD

**Issue by eamodio** [microsoft/vscode#124064]
I find the terminal active "details" button very useful (well the text at least -- I never click on it), but with left-aligned tabs it can be VERY far away from where my focus is.  Hopefully, there is a way to solve for that.

For my specific use-case, since I have my terminal split with another view

![image](https://user-images.githubusercontent.com/641685/118561910-c5886000-b739-11eb-88cf-bbafe5a47c1d.png)

It would be great if the content of the "details" button was duplicated into the terminal view's `titleDescription` like how the Timeline view shows the name of the active file

![image](https://user-images.githubusercontent.com/641685/118562067-05e7de00-b73a-11eb-9db7-dd2b0af8b5f2.png)

That would address my specific issue, but doesn't solve for the overall issue.

**Comment by Tyriar** [comment]
Looks a bit ugly with the repetition imo:

<img width="642" alt="Screen Shot 2021-05-18 at 4 56 53 AM" src="https://user-images.githubusercontent.com/2193314/118646895-9d2e4f00-b795-11eb-8096-c1861855e3b4.png">
<img width="690" alt="Screen Shot 2021-05-18 at 4 56 48 AM" src="https://user-images.githubusercontent.com/2193314/118646906-9f90a900-b795-11eb-9b5b-19d6e662469b.png">

**Comment by Tyriar** [comment]
Thoughts @meganrogge?

**Comment by meganrogge** [comment]
Yeah I think the duplication feels/looks heavy, but maybe we could make this happen only when the terminal tabs are left aligned?

**Comment by eamodio** [comment]
Hrm, I really like it, especially the one without the button
